app/src/main/res/layout/dialog_fragment_parking_lots.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please use just Constraint Layout for this exercise

Reply:
done